rct229/rulesets/ashrae9012019/section11/section11rule13.py

```diff
@@ -0,0 +1,152 @@
+                        building_segment["id"]
+                    ] = energy_required_to_heat_swh_use
+            return {
+                "energy_required_to_heat_swh_use_dict": energy_required_to_heat_swh_use_dict,
+                "service_water_heating_use_ids_dict": service_water_heating_use_dict,
+            }
+
+        class BuildingRule(PartialRuleDefinition):
+            def __init__(self):
+                super(Section11Rule13.RMDRule.BuildingRule, self).__init__(
+                    rmds_used=produce_ruleset_model_description(
+                        USER=False, BASELINE_0=True, PROPOSED=False
+                    ),
+                    manual_check_required_msg=APPLICABILITY_MSG,
+                )
+
+            def get_calc_vals(self, context, data=None):
+                building_b = context.BASELINE_0
+                energy_required_to_heat_swh_use_dict = data[
+                    "energy_required_to_heat_swh_use_dict"
+                ]
+                service_water_heating_use_dict = data[
+                    "service_water_heating_use_ids_dict"
+                ]
+                service_water_heating_info = {
+                    "btu_per_year": 0,
+                    "btu_per_sf_per_year": 0,
+                }
+                is_applicable = False
+                for building_segment in find_all("$.building_segments[*]", building_b):
+                    for swh_use in service_water_heating_use_dict[
+                        building_segment["id"]
+                    ]:
+                        if swh_use.get("use", 0.0) > 0:
+                            is_applicable = True
+                            energy_required_by_space = (
+                                energy_required_to_heat_swh_use_dict[swh_use["id"]][
+                                    building_segment["id"]
+                                ]
+                            )
+                            if None in energy_required_by_space.values():
+                                service_water_heating_info[
+                                    "btu_per_year"
+                                ] = "UNDETERMINED"
+                                service_water_heating_info[
+                                    "btu_per_sf_per_year"
+                                ] = "UNDETERMINED"
+                            else:
+                                service_water_heating_info["btu_per_year"] += sum(
+                                    energy_required_by_space.values()
+                                )
+                if service_water_heating_info["btu_per_year"] != "UNDETERMINED":
+                    floor_area = sum(
+                        space.get("floor_area", 0)
+                        for space in find_all(
+                            "$.building_segments[*]zones[*]spaces[*]", building_b
+                        )
+                    )
+                    if floor_area > 0:
```

Comment:
My general feeling is `floor_area` should not be 0. You can use `assert_(floor_area > 0, f"Floor area for building: {building['id']} is 0, check inputs."` to avoid this if statement.